Add documentation on cosmology to model conversion (#12622)
astropy/astropy: docs/modeling/physical_models.rst

    # Display plot
    plt.tight_layout(rect=[0, 0.03, 1, 0.95])
    plt.show()

.. _Cosmologies:

Cosmologies
===========

The instances of the |Cosmology| class (and subclasses) include
|Cosmology.to_format|, a method to convert a Cosmology to another python

> object. Relevant here is the conversion of any redshift method to a
> :class:`~astropy.modeling.FittableModel` instance, using the argument
> ``format="astropy.model"``.
> Each |Cosmology| :class:`~astropy.cosmology.Parameter` is converted to a
> :class:`astropy.modeling.Model` :class:`~astropy.modeling.Parameter`
> and the redshift-method to the model's ``__call__`` / ``evaluate``.
> Now you can fit cosmologies with data!

Reviewer: ```suggestion
object. Specifically, any redshift method can be converted to a
:class:`~astropy.modeling.FittableModel` instance using the argument
``format="astropy.model"``.
During the conversion, each |Cosmology| :class:`~astropy.cosmology.Parameter` is converted to a
:class:`astropy.modeling.Model` :class:`~astropy.modeling.Parameter`, while the redshift-method becomes the model's ``__call__`` / ``evaluate`` method.
This means cosmologies can now be fit with data!
```

The word choice made it difficult for me to follow on the first read. Above is my suggested reword.

Author: Thanks! I use this, but adjust the line lengths to match the rest of the document.

.. code-block::

    >>> from astropy.cosmology import Planck18

>     >>> model = Planck18.to_format("astropy.model", method="lookback_time")

Reviewer: ```suggestion
    >>> model = Planck18.to_format(format="astropy.model", method="lookback_time")
```
Add the keyword for consistency with proceeding paragraph and the conversion example below.

Author: 👍
